scripts/windows/build.bat

```diff
@@ -72,13 +84,18 @@ MKDIR granulate-utils 2> NUL
 COPY ..\requirements.txt requirements.txt
 FOR %%i in ("..\granulate-utils\setup.py" "..\granulate-utils\requirements.txt" "..\granulate-utils\README.md") do COPY "%%i" granulate-utils
 ECHO D | XCOPY ..\granulate-utils\granulate_utils .\granulate-utils\granulate_utils /E
+ECHO D | XCOPY ..\granulate-utils\glogger .\granulate-utils\glogger /E
 python -m pip install --no-cache-dir -r requirements.txt
 
 COPY ..\exe-requirements.txt exe-requirements.txt
 python -m pip install --no-cache-dir -r exe-requirements.txt
 MKDIR gprofiler\resources\python
 COPY ..\py-spy\py-spy.exe gprofiler\resources\python
+COPY ..\burn\burn.exe gprofiler\resources
 
 ECHO D | XCOPY ..\gprofiler .\gprofiler /E
 FOR %%i in ("..\pyi_build.py" "..\pyinstaller.spec") do COPY "%%i" .
 pyinstaller pyinstaller.spec
+
+:clear_error
```

Comment:
Need to check if pyinstaller susceeds before `clear_error`?

Reply:
Yes.